pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -772,7 +933,7 @@ async def parse_packages(self, primary_xml, filelists_xml, other_xml):
         )
 
         # skip SRPM if defined
-        if "srpm" in self.skip_types:
+        if "srpm" in self.skip_types and not self.mirror:
```

Comment:
It's good that we ignore skip_types, however I think user experience is confusing if they specified skip_types option and still got all the content. Not sure what to do about it, apart from having a separate viewset/repo for mirror.

Reply:
I agree.  Probably we should not allow this option if `mirror=True`

Reply:
@goosemania https://github.com/pulp/pulp_rpm/pull/1994/files#diff-37aef788d3a12e65873c374ff05bc8629a8b636cab51ac31c127f94d43f1314eR138

Reply:
Well, I guess that would technically be a breaking change, but perhaps a sensible one?  I'm not sure what our tolerance here should look like.